en/gps_compass/rtk_gps_gem1305.md

```diff
@@ -0,0 +1,144 @@
+- High performance antenna
+
+## Frequencies
+
+- GPS/QZSS: L1 C/A, L5C
+- GLONASS: L1OF
+- BEIDOU: B1I, B2a
+- GALILEO: E1, E5a
+- IRNSS: L5
+
+## GNSS
+
+- 128 hardware channels
+- 3D accuracy: **1.5m** CEP
+- RTK accuracy: **2cm** +1PPM(H), 3cm+1PPM(V)
+
+## Interface
+
+- UART: 230400bps default
+- SMA connector for antenna
+- Output rate 5Hz default, up to 10Hz
+- Main power supply: 4.7~5.2V
+
+## Protocol
+
+- NMEA-0183 output
+- RTCMv3 input/output
+
+## Environment
+
+- Operating temp. -20~85°C
+
+## Dimension and Weight
+
+- 55x55x12mm
+- 50g (GEM1305) 26g (NANO with Helix)
+
+## Pinout
+
+The board is connected to the autopilot via UART interface.
+
+![GEM1305 connector](../../assets/hardware/gps/datagnss_gem1305/gem1305_connector.png)
+
+The 1.25mm pitch 6P connector (from left: PIN1 to PIN6):
+
+- 1: GND
+- 2: NC
+- 3: PPS
+- 4: Rx
+- 5: Tx
+- 6: 5V
+
+The board does not include a compass and so does not require connections for a compass part.
+
+## Hardware Setup
+
+RTK requires a base RTK module attached to the ground station, and a rover RTK module on the vehicle.
+The data from the base needs to be transmitted to the drone via telemetry radio and inputed into the RTK receiver on the rover.
+
+![RTK setup overview](../../assets/hardware/gps/datagnss_gem1305/setup_overview.png)
+
+The setup/connections for base and rover modules are shown below.
+
+### Base Setup (GCS)
+
+The base station connections are shown in the diagram below, along with the connection to a telemetry radio.
+
+![Base module setup](../../assets/hardware/gps/datagnss_gem1305/base_gnss_setup.png)
+
+Note that for the base we recommend the [NANO RTK Receiver](https://www.datagnss.com/collections/gnss-for-drone/products/multi-band-rtk-receiver-package) because it is easier to setup.
+
+<img src="../../assets/hardware/gps/datagnss_gem1305/nano_rtk_with_case.png" width="500px" alt="DATAGNSS NANO RTK Receiver">
+
+See to [How to setup Base station](https://wiki.datagnss.com/index.php/GEM1305-autopilot) for more details.
+
+### Rover Setup (PX4)
+
+The rover setup showing connections to a GPS port and the (required) connection to a telemetry radio are shown in the diagram below.
+
+![Rover module and pixhawk big picture connections](../../assets/hardware/gps/datagnss_gem1305/rover_gnss_setup.png)
+
+The diagram below shows how you might connect to the `GPS` port on the Pixawk 2.4.8 flight controller.
+The same connection approach might be used on a more recent flight controller.
+
+::: warning
+Pixawk 2.4.8 is a clone of a very old Pixhawk series design.
+Even though it is inexpensive it is not recommended: the board has such little memory that it can be difficult to find space for all the software components you need.
+It also uses non-Pixhawk standard connectors which are far less reliable, and works with many fewer components out of the box.
+:::
+
+![Rover module setup](../../assets/hardware/gps/datagnss_gem1305/pixhawk2_4_8_connections.png)
```

Comment:
Am I correct in thinking that this comes with a cable for Pixhawk 2.4.8? Have you heard of the Pixhawk connector standard? 

This is a very very old flight controller, commonly used only by hobbyists for whom the time getting the firmware to fit into the small flash space is worth the cost. Many PX4 consumers use Pixhawk 5 or 6 and might not buy this because of the hassle of creating a new cable. Do you supply it with the Pixhawk standard connector?

Reply:
We tested it on Pixhawk 4 and Pixhawk 6. We provide the standard cables for pixhawk with the receiver.

Reply:
Hi @hamishwillee, Thanks.
Don't worry about this issue, we provide more than one cable to meet the needs of different devices.

Reply:
The schematic we provide is only for illustrating the connection method. I think this warning is unnecessary. We can update the Pix6 schematic and also emphasize that we provide cables compatible with the Pixhawk standard along with the device.

Reply:
Absolutely, if you can provide an image using a pixhawk standard compatible flight controller, then we don't need this.

We can refer to the cables provided by their pixhawk standard name too. I see in package list you do state GH-1.25mm-6P cable

Reply:
Hi @hamishwillee , I attached a new picture to show the GEM1305 connect the Pix6.
![GEM1305-connect-pix6](https://github.com/user-attachments/assets/2b384361-9834-4034-89a7-bf34ec3f3d48)

Reply:
That's nice. Thank you.